app/lib/frontend/templates.dart

```diff
@@ -463,11 +463,18 @@ class TemplateService {
     );
     final content = _renderTemplate('pkg/show', values);
     final versionString = isVersionPage ? '${selectedVersion.id} ' : '';
+    var pageDescription =
+        '${selectedVersion.package} ${selectedVersion.version}';
+    if (isFlutterPlugin) {
```

Comment:
Nit: This name is a bit misleading. What we want to capture (and do) are Flutter packages -- ie. packages that depend on Flutter. A Flutter plugin is a special kind of Flutter package that also contains native code. `isFlutterPackage` would be a better name for this variable.

Reply:
I'll update that in a subsequent PR. It has a bit of history, and both the name and its initialization could be updated.